test/verify/check-login

```diff
@@ -622,9 +622,9 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
             # https://bugzilla.redhat.com/show_bug.cgi?id=1836182
             self.allow_journal_messages('.*type=1400.*avc:  denied  { chown }.*comm="cockpit-session".*')
 
-        if module == 'pam_tally2':
```

Comment:
Is it necessary to remove the check?  If we do a `restore_file()` on the logfile, that means that it should be removed after this test.  We also restore the PAM config, so nothing else should create it either...  It might be a good way to make sure "nothing unexpected" is going on here...

Reply:
I didn't see the test fail under Fedora, so it's not strictly necessary. I just thought it'd be both more symmetric and more robust to backup/remove/restore a possibly existing /run/faillock/admin as well, as the test bed may also already have it. *Our* test VMs don't, but we also run these tests on QE/beaker etc. machines. If you prefer, I can put back the `if`, and we only drop it if we run into trouble in downstream tests.